packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Role Assignment
+---
+<ApiLifecycle access="beta" />
+
+# Role Assignment
+Role assignment to principals makes them administrators of your org.
```

Comment:
Add another sentence: Principals can be users or groups of users.

Reply:
added